spec/facility_spec.rb

```diff
@@ -17,10 +25,159 @@
   describe '#add service' do
     it 'can add available services' do
       expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
+      @facility.add_services('New Drivers License')
+      @facility.add_services('Renew Drivers License')
+      @facility.add_services('Vehicle Registration')
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
+    
+    it "can add available services to specific facility" do
+      facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+      facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
+      facility_1.add_services('Vehicle Registration')
+      
+      expect(facility_1.services).to eq(['Vehicle Registration'])
+    end
+  end
+
+  describe '#registers vehicle' do
+    it "registers vehicle at specific facility" do
+      facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+      facility_1.add_services('Vehicle Registration')
+      
+      facility_1.register_vehicle(@cruz)
+      expect(@cruz.registered?).to eq(true)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+
+      facility_1.register_vehicle(@camaro)
+      expect(@camaro.registered?).to eq(true)
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@camaro.plate_type).to eq(:antique)
+
+      facility_1.register_vehicle(@bolt)
+      expect(@bolt.registered?).to eq(true)
```

Comment:
Nice job testing `.registered?` here. This would be considered and Integration test. You also need to Unit test this method in your `vehicle_spec`